model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -45,20 +44,20 @@ class FindingsMatcherTest : WordSpec() {
 
     override fun isInstancePerTest() = true
 
-    private fun setupLicenseFinding(license: String, path: String, line: Int = Random.nextInt(1..1000)) {
+    private fun setupLicenseFinding(license: String, path: String, line: Int = 2) {
         licenseFindings.add(
             LicenseFinding(
                 license = license,
                 location = TextLocation(
                     path = path,
                     startLine = line,
-                    endLine = line
+                    endLine = line + 1
```

Comment:
Similarly here, why does it make sense to *always* make `endLine = startLine + 1`? Is there was a requirement that license statements always need to span multiple lines (i.e. at least 2 lines),  we should document this here. If there is no such requirement, why not simply leave it at `endLine = line`, also to match the `setupCopyrightFinding` code below?